### ISSUE TITLE
how to ensure the iou prediction between zero and one?

### ISSUE THREAD

**Issue by Altria7** [yformer/EfficientSAM#77]
Hello, I am a student specializing in deep learning. What confuses me is that the generation of IOU predictions does not involve the use of activation functions like sigmoid or softmax. How can we ensure that the output results are between zero and one? Additionally, how do you calculate the loss between IOU predictions and the actual IOU when fine-tuning your model? Thank you.

**Comment by yformer** [comment]
Hi @Altria7, thanks for your interest in our work! We have a sigmoid to map the output prediction between one and zero. 

Wonder if you are interested in our Efficient Track Anything for unified image and video segmentation, https://github.com/yformer/EfficientTAM
